travel/migrations/0007_auto_20160310_1914.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.4 on 2016-03-10 19:14
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('travel', '0006_auto_20160309_1856'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
This migration can be simplified. All the `RemoveField` operations are unnecessary, as `DeleteModel` is being run on eventlocationprincipaltravel anyway.